src/Biryuchkov/BiryuchkovTask3/normals.h

```diff
@@ -0,0 +1,17 @@
+#ifndef NORMALS_H
+#define NORMALS_H
+#include <QOpenGLShaderProgram>
+#include <vector>
+
+class Normals
+{
+public:
+    Normals();
+    void setNormal(GLfloat _x,GLfloat _y,GLfloat _z);
+    void setNormal(const std::vector<GLfloat>& vec);
+    const std::vector<GLfloat>& getNormal();

```

Comment:
```suggestion
    [[nodiscard]] const std::vector<GLfloat>& getNormal() const noexcept;

```

Поправьте в остальных местах сами, больше не буду писать :)